src/lib/y2partitioner/widgets/help.rb

```diff
@@ -12,216 +12,119 @@ def included(_target)
+
+        fc_wwpn:          N_("<b>WWPN</b> shows the World Wide Port Name for\nFibre Channel " \
+          "disks."),
+
+        file_path:        N_("<b>File Path</b> shows the path of the file for\nan encrypted " \
+          "loop device."),
+
+        format:           N_("<b>Format</b> shows some flags: <tt>F</tt>\nmeans the device " \
+              "is selected to be formatted."),
+
+        fs_id:            N_("<b>FS ID</b> shows the file system id."),
+
+        fs_type:          N_("<b>FS Type</b> shows the file system type."),
+
+        label:            N_("<b>Label</b> shows the label of the file\nsystem."),
+
+        lvm_metadata:     N_("<b>Metadata</b> shows the LVM metadata type for\nvolume groups."),
+
+        model:            N_("<b>Model</b> shows the device model."),
+
+        mount_by:         N_("<b>Mount by</b> indicates how the file system\nis mounted: " \
+            "(Kernel) by kernel name, (Label) by file system label, (UUID) by\n file system " \
+            "UUID, (ID) by device ID, and (Path) by device path.\n"),
+
+        mount_point:      N_("<b>Mount Point</b> shows where the file system\nis mounted."),
+
+        num_cyl:          N_("<b>Number of Cylinders</b> shows how many\ncylinders the hard disk " \
+          "has."),
+
+        parity_algorithm: N_("<b>Parity Algorithm</b> shows the parity\nalgorithm for RAID " \
+          "devices with RAID type 5, 6 or 10."),
+
+        pe_size:          N_("<b>PE Size</b> shows the physical extent size\nfor LVM volume " \
+        "groups."),
+
+        raid_version:     N_("<b>RAID Version</b> shows the RAID version."),
+
+        raid_type:        N_("<b>RAID Type</b> shows the RAID type, also\ncalled RAID level, " \
+        "for RAID devices."),
+
+        size:             N_("<b>Size</b> shows the size of the device."),
+
+        start_cyl:        N_("<b>Start Cylinder</b> shows the start cylinder\nof the partition."),
+
+        stripes:          N_("<b>Stripes</b> shows the stripe number for LVM\nlogical volumes " \
+          "and, if greater than one, the stripe size in parenthesis.\n"),
+
+        type:             N_("<b>Type</b> gives a general overview about the\ndevice type."),
+
+        udev_id:          N_("<b>Device ID</b> shows the persistent device\nIDs. This field " \
+        "can be empty."),
+
+        udev_path:        N_("<b>Device Path</b> shows the persistent device\npath. This field " \
+        "can be empty."),
+
+        used_by:          N_("<b>Used By</b> shows if a device is used by\ne.g. RAID or LVM. " \
+              "If not, this column is empty."),
+
+        uuid:             N_("<b>UUID</b> shows the Universally Unique\nIdentifier of the file " \
+          "system."),
+
+        vendor:           N_("<b>Vendor</b> shows the device vendor.")
+      }.freeze
+
+      # help texts appended to common help only in Mode.normal
+      NORMAT_MODE_TEXTS = {
```

Comment:
well, I let formatting on rubocop, but old version we use for yast do not do it. So which one you prefer?